roofit/roofitcore/src/RooFormula.h

```diff
@@ -25,6 +25,8 @@
 class RooAbsReal;
 
 class RooFormula : public TNamed, public RooPrintable {
+   friend class RooFormulaVar; // for accessing TFormula from ::translate.
+   friend class RooGenericPdf; // for accessing TFormula from ::translate.
```

Comment:
We could narrow it down to the `translate` function, however, we will have to include RooFormulaVar.h and RooGenericPdf.h which might be a layering violation...

Reply:
I think it would be better here to have a public accessor for the `_tFormula` member that is used in the `translate()` function. We can change the public interface as we like because the `RooFormula` header is not public anyway, and like this we don't need friend classes.

Reply:
Do you prefer "leaking" the implementation detail TFormula? I am happy to do it but that means that it could be used everywhere else for random things ;)

Reply:
Yes, that's okay here for me, as it only leaks it to the rest of RooFit and not the user